screensaver.bing/resources/skins/default/1080i/default.xml

```diff
@@ -46,8 +53,17 @@
                     <width>1920</width> 
                     <height>1080</height> 
                     <aspectratio scalediffuse="false">scale</aspectratio>
+                    <animation effect="zoom" start="110" end="130" center="auto" time="25000" tween="cubic" easing="inout" condition="StringCompare(Window(10000).Property(bing_animation),okay)" pulse="true">Conditional</animation>
```

Comment:
FYI you can leave the window id empty. In your instance of WindowXMLDialog just use `self.setProperty(key,value)`. On your xml file use `Window().Property(bing_animation)`. I am pretty much doing the same since I found odd to set properties on the home window all the time.